log-replication/resources/corfu_replication_config.properties

```diff
@@ -0,0 +1,18 @@
+log_reader_max_retry=5
+log_reader_queue_size=1
+log_reader_resend_timer=500
+log_reader_error_on_message_timeout=true
+
+log_writer_queue_size=20
+log_writer_ack_cycle_count=1
+log_writer_ack_cycle_time=100
+
+primary_site="primary site"
```

Comment:
Is this info supposed to be here?... I'm confused as to what this file has.. 

This info is supposed to come from SiteManager.. and other configs not sure how we should refer to them... 

Also we have a file for the streams to replicate, which name does that file have? 

@pankti-m could you help clarify/unify the different configurations?